src/main/java/teammates/storage/search/SearchManager.java

```diff
@@ -222,13 +218,15 @@ List<T> convertDocumentToAttributes(List<SolrDocument> documents) {
 
         for (SolrDocument document : documents) {
             T attribute = getAttributeFromDocument(document);
-            if (attribute == null) {
-                // search engine out of sync as SearchManager may fail to delete documents
-                // the chance is low and it is generally not a big problem
-                String id = (String) document.getFirstValue("id");
-                deleteDocuments(Collections.singletonList(id));
-                continue;
-            }
+            // Disabled for now
```

Comment:
Is this chunk still relevant?

Reply:
based on @samuelfangjw's comment [here](https://github.com/TEAMMATES/teammates/pull/12340/files/8616b9274578d9be37b418152f25555142d64e7d#r1179414185) to disable it for now... no harm in leaving it in first, i think we can clean up the comments after migration (i suppose after migration this file would be deleted)

for the new search manager, i re-enabled it since there were times when the search engine was out of sync, resulting in it finding documents that were already deleted